### ISSUE TITLE
No focus ring on "No Payloads" view in payload library

### ISSUE THREAD

**Issue by erikberglund** [erikberglund/ProfileCreator#26]
When no payload is present in the payload library, the "No Payloads" view is shown.

If you drag a payload from the profiles part back down on the library part, no focus ring is shown when hovering that view. This should be fixed.
